roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -25,6 +25,7 @@ global
   daemon
   pidfile /var/run/haproxy/haproxy.pid
   tune.bufsize {{ haproxy.bufsize }}
+  stats socket /var/run/haproxy/stats.sock mode 600
```

Comment:
Have we verified this doesn't conflict with the HTTP stats in the default stanza below?